agent/consul/multilimiter/multilimiter.go

```diff
@@ -145,14 +155,45 @@ func (m *MultiLimiter) Allow(e LimitedEntity) bool {
 			config = prefixConfig
 		}
 	}
-
 	limiter := &Limiter{limiter: rate.NewLimiter(config.Rate, config.Burst)}
```

Comment:
I wonder if constructing new limiters in `runStore` would result in fewer allocations under heavy load?

It might be a micro-optimization, but because we flush new writers every 10ms we'll likely end up creating limiters for the same key over and over.

Reply:
We need the limiter to allow the current request, We can still return true but  I don't think that this optimization is necessary.